sql/core/src/test/resources/sql-tests/results/postgreSQL/text.sql.out

```diff
@@ -312,7 +312,7 @@ org.apache.spark.sql.AnalysisException
   "errorClass" : "INVALID_PARAMETER_VALUE",
   "sqlState" : "22023",
   "messageParameters" : {
-    "expected" : "expects %1$, %2$ and so on, but got %0$.",
+    "value" : "expects %1$, %2$ and so on, but got %0$.",
```

Comment:
This is why it is named `expected` - the parameter tells what Spark expects. cc @srielau